x-pack/solutions/search/plugins/search_playground/public/components/query_mode/query_output.tsx

```diff
@@ -25,10 +32,14 @@ export const ElasticsearchQueryOutput = ({
   queryResponse,
   isError,
   queryError,
+  isLoading,
 }: ElasticsearchQueryOutputProps) => {
```

Comment:
one thing noticed is useQuery will retry 3 times by default. We should check if its a 400 error then return the error message without retry. 

https://github.com/user-attachments/assets/2a4b24b7-7dbc-4a4e-be46-dcdb696b8fe9